src/guide/jbs-jdk-bug-system.md

```diff
@@ -18,111 +18,81 @@ openjdk.org/).
 
 ### Implementing a JEP
-It is recommended that for [JEP]{.jbs-field}s that the implementation is spread across one or more [Enhancement]{.jbs-field}s as above.
+
+It's recommended for [JEP]{.jbs-value}s that the implementation is spread across one or more [Enhancement]{.jbs-value}s as described above.
 
 ## Issue states
 
-JBS only has a few states in which an issue can be in:
+JBS only has a few states in which a bug or enhancement can be in:
 
-<table class="dictionary" summary="JBS Label Dictionary">
-  <tr style="text-align:left;"><th>Type</th><th>Covers</th></tr>
-<tr>
-    <td class="dictionary">[New]{.jbs-field}</td>
-    <td class="dictionary">Initial state after an issue is filed. [Bugs]{.jbs-field} in the JDK Project must be taken out of the [New]{.jbs-field} state (Triaged - see below) in a timely manner. In general, this is recommended to be done for all issue types and projects as a sign that the issue is correctly filed, and will be seen by the right group - this is especially important towards the end of a release.</td>
-</tr>
-<tr>
-    <td class="dictionary">[Open]{.jbs-field}</td>
-    <td class="dictionary">Once the issue has been triaged it will be in the [Open]{.jbs-field} state and will stay here until the assignee decides to work on it, at which point it is encouraged that the "Start Work" option be selected to move it to [In Progress]{.jbs-field}.</td>
-</tr>
-<tr>
-    <td class="dictionary">[In Progress]{.jbs-field}</td>
-    <td class="dictionary">The [In Progress]{.jbs-field} state has the option of the sub-states ([Understanding]{.jbs-field}): [Cause Known]{.jbs-field}, [Fix Understood]{.jbs-field}, [In Review]{.jbs-field}.</td>
-</tr>
-<tr>
-    <td class="dictionary">[Closed]{.jbs-field}/[Resolved]{.jbs-field}</td>
-    <td class="dictionary">While [Closed]{.jbs-field} and [Resolved]{.jbs-field} are essentially equivalent, when it comes to fixing issues there is an additional [Verify]{.jbs-field} step available between the [Resolved]{.jbs-field} and [Closed]{.jbs-field} states.</td>
-</tr>
-</table>
+| Type | Covers |
+|:-|:----------|
+| [New]{.jbs-value} | Initial state after an issue is filed. Bugs in the JDK Project must be taken out of the [New]{.jbs-value} state ("Triaged" - see below) in a timely manner. In general, triaging is recommended to be done for all issue types and projects as a sign that the issue is correctly filed, and will be seen by the right developers - this is especially important towards the end of a release. |
+| [Open]{.jbs-value} | Once the issue has been triaged it will be in the [Open]{.jbs-value} state and will stay here until the assignee decides to work on it, at which point it's encouraged that the "Start Work" option be selected to move it to [In Progress]{.jbs-value}. |
```

Comment:
Recommend "an assignee".  "The assignee" implies that a bug has been assigned during triage, which is not always the case.  There are many issues in the system which don't have an assignee and that's not considered a problem.

Reply:
Agreed. Fixed.